src/ts/test/helpers/prosemirror.ts

```diff
@@ -15,3 +21,21 @@ const build = builders(schema, {
 
 export const createDoc = build.doc;
 export const p = build.p;
+
+/**
+ * We cannot compare document decorations to plugin decorations, as the
+ * .ProseMirror-widget class is applied – but we can compare their specs.
```

Comment:
I still feel like I'm lacking knowledge in this area, and I would be interested to learn more about how decorations are applied in PM.